synapse/appservice/api.py

```diff
@@ -227,7 +227,9 @@ async def push_bulk(
 
         try:
             await self.put_json(
-                uri=uri, json_body=body, args={"access_token": service.hs_token},
+                uri=uri,
+                json_body=body,
+                args={"access_token": service.hs_token},
```

Comment:
I was surprised by this change, but I think it is due to:

> added support for pre-splitting collections by putting an explicit trailing comma inside

From the [black changelog](https://github.com/psf/black/blob/master/CHANGES.md).

If you remove the trailing comma than black won't reformat this since it is on a short enough line. I don't have a strong opinion about whether we would want to remove the commas instead of reformatting. 🤷 

This seems to make up most of the diff fwiw.

Reply:
We're making a change either way. I'd prefer not to fix them all 😅

Is there a way to retry the build? It looks like it failed on one of the runs, https://buildkite.com/matrix-dot-org/synapse/builds/15308

Reply:
I've retried it for you. You should see a Retry button if you're signed in to Buildkite and have the appropriate permissions. If you don't have the latter, ask in the CI Internal room.

Reply:
Thanks @anoadragon453! Looks all ✅ now. I've asked in https://matrix.to/#/!QaKLeuyXEEnDvFiPex:matrix.org/$1613508293261011LJpKF:matrix.org?via=matrix.org&via=sw1v.org&via=amorgan.xyz